### PR TITLE
ENT-3109: Create Jenkins job for translation extraction

### PR DESCRIPTION
**Pull request by sonalidhome**
(gettext/msgattrib)
 - Added a Gradle task to perform msgattrib on po files.

### CODE REVIEW

**Comment by sonalidhome** [comment]
Note: Once this gets approved and merged, we need to cherry-pick this into candlepin-3.1-HOTFIX and candlepin-2.9-HOTFIX branches.

File: buildSrc/src/main/groovy/Gettext.groovy [hunk 1]
```diff
@@ -108,5 +108,27 @@ class Gettext implements Plugin<Project> {
                 }
             }
         }
+
+        def msgattrib_task = project.task('msgattrib') {
+            description = 'The msgattrib program filters the messages of a translation catalog according to their attributes, and manipulates the attributes.'
```
Comment (nikosmoum) [inline buildSrc/src/main/groovy/Gettext.groovy:113]: Even though this is the description of what msgattrib is doing in general, we're using msgattrib in a very specific way here, so let's change this description to reflect that. Something like `Use msgattrib to remove obsolete strings (that were already removed from the source code & template file) from translation files. ` should be good

**Review by nikosmoum — APPROVED** [review]
ACK (only 1 minor change needed)

File: buildSrc/src/main/groovy/Gettext.groovy (hunk 1 above)
Comment (sonalidhome) [reply buildSrc/src/main/groovy/Gettext.groovy]: Updated.

**Pull request merged**